### ISSUE TITLE
cannot add

### ISSUE THREAD

**Issue by ghost** [mozfet/meteor-autoform-materialize-modals#3]
this version is incompatible with materialize 3.0.3 

while selecting package versions:
error: Conflict: Constraint mozfet:autoform-materialize@2.0.10 is not satisfied by mozfet:autoform-materialize 3.0.3

any idea?

**Comment by mozfet** [comment]
I'm stumped. There is no direct dependancies in mozfet:autoform-materialize on mozfet:autoform-materialize-modals and visa versa. Heck there is not even a dependancy on Materialize...

**Comment by mozfet** [comment]
I am using the latest versions in a production system without any issues...

.meteor/versions
```
mozfet:autoform-materialize@3.0.3
mozfet:autoform-materialize-modals@1.0.8
```

package.json
```
  "materialize-css": "^0.100.2"
```

**Comment by mozfet** [comment]
I suggest a clean build. Delete versions and local folder in meteor folder... and try again...

**Comment by ghost** [comment]
Yup,  after check the playground,  version 1.0. 8 installed...

**Comment by mozfet** [comment]
You message states:  version 2.0.10 not compatable with 3.0.3 - if your version is ^2.0.2 then that is true, its a major version upgrade and not compatible with the old version. And thus when doing meteor update you have to update it manually.

**Comment by ghost** [comment]
Just to make it clear,  now my meteor version is 1.5.2 or the latest version,  and my mozfet materialize is 3.0.3, when i tried to add modal from meteor packages,  then i got that error says version 2.0.2 are not compatible with materialize 3.0.3,  so i install the 1.0.8 version and it success...  But i dont try it yet...  So now,  im using the 1.0.8 version

**Comment by mozfet** [comment]
Now I am really confused too many version numbers flying around... please list the before versions and the after versions carefully.

**Comment by ghost** [comment]
ok so first thing first, 
before i add materilaize modal, i have
1. Meteor version 1.5.2
2. autoform-materiliazie 3.0.3

then try to add meteor add mozfet:autoform-modals-materialize, as per document said, but the autoform-materialize-modals 2.0.2 incompatible with autoform-materialize 3.0.3, so i installed the autoform-materialize-modal 1.0.8,

so right now i have
1. Meteor version 1.5.2
2. autoform-materialize 3.0.3
3. autoform-materialize-modal 1.0.8 (i dont use it yet, so i dont know if its working or not)

i hope its clear enough...

**Comment by mozfet** [comment]
perfect... I see the problem, but do not know how you got there... there is no autoform-materialize-modals 2.0.2 ... latest version is 1.0.8...

so you have the latest versions of everything... Except AutoForm 6.2... which I have not tested with yet...

**Comment by ghost** [comment]
sorry forgot to mention that i have autoform 6.2, the latest one..... to reproduce the error, just install the latest one, from autoform 6.2, autoform-materialize 3.0.3, then try meteor add mozfet:autoform-modals-materialize, then you'll get the error. BTW, what is the different betwen this two :
 
https://atmospherejs.com/mozfet/autoform-modals-materialize
https://atmospherejs.com/mozfet/autoform-materialize-modals

one is 1.0.8, and anothe one is 2.0.10??..

i see the problem... in this repo you tell us about how to add the package with this link:
https://atmospherejs.com/mozfet/autoform-modals-materialize

then explain about how to use it part, you use this link:
https://atmospherejs.com/mozfet/autoform-materialize-modals

which is different... i dont know why...

**Comment by mozfet** [comment]
hmmmm... I should I have updated Atmosphere when I depricated autoform-modals-materialize. I inherited the old one from the previous package manager which used a seperate modal framework, but had to redo it completely as Materialize improved modals. So I decided to make a new package and align the naming, and to leave to old package for phasing out. I will mark it as depricated and post a link to the right repo. 

Thus to be clear: Do not use mozfet:autoform-modal-materials, it is depricated.
